Lib/test/test_py3kwarn.py

```diff
@@ -206,6 +206,21 @@ def test_frame_attributes(self):
                 w.reset()
                 self.assertWarning(setattr(f, attr, None), w, expected)
 
+    def test_listiter_next(self):
+        expected = "the attribute 'next' is not supported in 3.x: use '__next__' or create a 'next' alias"
```

Comment:
I think we mean "The global function `next`"?

Reply:
The global builtin works:

py2.x:

```
>>> marks = [65, 71, 68, 74, 61]
>>> iterator_marks = iter(marks)
>>> marks_1 = next(iterator_marks)
>>> iterator_marks.next()
71
```

py3.x:

```
>>> marks = [65, 71, 68, 74, 61]
>>> iterator_marks = iter(marks)
>>> marks_1 = next(iterator_marks)
>>> iterator_marks.next()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'list_iterator' object has no attribute 'next'
```

its the `internext()` that fails:

`x = [2, 3]`

This works:

`next(iter(x))`

This wont:

`iter(x).next()`

unless its changed to:

`iter(x).__next__()`

Reply:
Oh, I see!